### PR TITLE
Update GOVERNANCE policy from upstream

### PR DESCRIPTION
**Pull request by Starefossen**
**Notable changes:**
- Document updated from upstream [Working Group Bootstrap Governance](https://github.com/nodejs/io.js/blob/master/WORKING_GROUPS.md#bootstrap-governance)
- Added Developer's Certificate of Origin 1.0
- Added Code of Conduct
- Changes to the WG membership follows standard consensus seeking process

```
The WG may add additional members to the WG by unanimous consensus.

A WG member may be removed from the WG by voluntary resignation, or by
unanimous consensus of all other WG members in an issue or pull request
on the [nodejs/docker-iojs](https://github.com/nodejs/docker-iojs) repository
```

Has been replaced with 

```
The WG may add, or remove, members to and from the WG. A WG member may
be choose to be removed from the WG by voluntary resignation.

Changes to WG membership should be posted in the
[nodejs/docker-iojs](https://github.com/nodejs/docker-iojs) repository
as an issue or pull request with the ***WG-agenda*** label followed by
the consensus seeking process described below.
```

### CODE REVIEW

**Comment by retrohacker** [comment]
Looks fantastic :+1: thanks for this!

File: GOVERNANCE.md [hunk 1]
```diff
@@ -63,14 +62,16 @@ make decisions efficiently.
 There is no specific set of requirements or qualifications for WG
 membership beyond these rules.
 
-The WG may add additional members to the WG by unanimous consensus.
+The WG may add, or remove, members to and from the WG. A WG member may
+be choose to be removed from the WG by voluntary resignation.
```
Comment (pesho) [inline GOVERNANCE.md]: "may be choose" -> "may choose"

Comment (Starefossen) [inline GOVERNANCE.md]: Fixed!

**Comment by pesho** [comment]
LGTM, thanks :+1:

**Comment by Starefossen** [comment]
Ping @nodejs/docker

**Comment by retrohacker** [comment]
I think this is safe to merge, enough time has passed for anybody to raise objections and the wording is in line with that of io.js project.

**Comment by Starefossen** [comment]
Merged :sunglasses:

**Comment by retrohacker** [comment]
:heart:

**Pull request merged**